### ISSUE TITLE
Using stylelint-config-standard -> at-rule-no-unknown

### ISSUE THREAD

**Issue by enjikaka** [stylelint/stylelint#1565]
Fresh install of stylelint and stylelint-config-standard gives a "at-rule-no-unknown" error.

**Comment by evilebottnawi** [comment]
@enjikaka please format your issue as template

**Comment by enjikaka** [comment]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where fresh install of stylelint and stylelint-config-standard gives a `
Undefined rule "at-rule-no-unkown"` error.

> Which rule, if any, is this issue related to?

at-rule-no-unknown

> What CSS is needed to reproduce this issue?

none

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "extends": "stylelint-config-standard"
}
```

> Which version of stylelint are you using?

Latest as of right now:
"stylelint": "^6.8.0",
"stylelint-config-standard": "^10.0.0",

> How are you running stylelint: CLI, PostCSS plugin, Node API?

cli

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no

> What did you expect to happen?

No error

> What actually happened (e.g. what warnings or errors you are getting)?

Undefined rule "at-rule-no-unkown"

```shell
test.css
1:21    Expected 2 spaces (indentation)
```

@evilebottnawi

**Comment by jeddy3** [comment]
@enjikaka Can you please double check you're running the latest version of both as I wasn't able to replicate this? 

Can you please try:

```console
stylelint --version
./node_modules/.bin/stylelint --version
```
 
And let us know what versions you get back.

**Comment by enjikaka** [comment]
@jeddy3 The first one tells me 6.2.2. Maybe it's installed globally.

**Comment by evilebottnawi** [comment]
@enjikaka try to update `stylelint` globally

**Comment by jeddy3** [comment]
That'll be why.  You can update it with `install -g stylelint@latest`... or you might want to remove the globally installed version with `uninstall -g stylelint` so this doesn't trip you up again.

**Comment by enjikaka** [comment]
`npm update stylelint --global` and a restart of VS Code did not solve the issue.

**Comment by evilebottnawi** [comment]
@enjikaka show you command how you run `stylelint`

**Comment by enjikaka** [comment]
@evilebottnawi Works via the command. The error only occurs in VS Code with the Stylelint extension linked on the site of Stylelint.

**Comment by evilebottnawi** [comment]
@enjikaka `Undefined rule "at-rule-no-unkown" error.` in rule name error -> `at-rule-no-unknown`, maybe you disable rule or change config somewhere?

**Comment by m-allanson** [comment]
Hi @enjikaka, `at-rule-no-unknown` was just added in stylelint v6.8.0. I think your issue is caused by the VS Code `stylelint` extension running on an older version of stylelint.

It looks like `vscode-stylelint` [uses a package called `stylelint-vscode`](https://github.com/shinnn/vscode-stylelint/blob/master/package.json). `stylelint-vscode` in turn [uses `stylelint` v6.0.3](https://github.com/shinnn/stylelint-vscode/blob/master/package.json#L40)

To use `at-rule-no-unknown` in VS Code the following would need to happen:

- [`stylelint-vscode`](https://github.com/shinnn/stylelint-vscode) would need to update its `stylelint` dependency to v.6.8.0
- [`vscode-stylelint`](https://github.com/shinnn/vscode-stylelint/) would need to update its `stylelint-vscode` dependency to the new version

**Comment by jeddy3** [comment]
`vscode-stylelint` seems to use a [caret range](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004) of `^6.0.3` though, and so everything up to  `7.0.0` is covered.

I don't know anything about vscode extensions, but there might be a dependency update option with vscode or you could try uninstall and reinstalling the extension.

**Comment by jeddy3** [comment]
> Works via the command.

@enjikaka As this works via the CLI, it's likely a issue with an outdated dependency in your vscode extension. If the suggestions above haven't resolved the issue for you, then perhaps raise a new issue in the `vscode-stylelint` repo. Good luck.
